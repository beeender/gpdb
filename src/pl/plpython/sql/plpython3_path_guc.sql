--
-- Tests for functions that return set PYTHONPATH guc only for gpdb6 plpython3
--
LOAD 'plpython3';

show plpython3.python_path;

set plpython3.python_path='/foo';

show plpython3.python_path;

set plpython3.python_path='/bar';

-- Set up test path functions first.
CREATE FUNCTION test_path_added() 
RETURNS text AS $$
  import sys
  return str(sys.path[0])
$$ language plpython3u;

SELECT test_path_added();
