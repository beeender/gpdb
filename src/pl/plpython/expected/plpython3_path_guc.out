--
-- Tests for functions that return set PYTHONPATH guc only for gpdb6 plpython3
--
LOAD 'plpython3';
show plpython3.python_path;
 plpython3.python_path 
-----------------------
 
(1 row)

set plpython3.python_path='/foo';
show plpython3.python_path;
 plpython3.python_path 
-----------------------
 /foo
(1 row)

set plpython3.python_path='/bar';
ERROR:  SET PYTHONPATH for plpython3 can only set once in one session
-- Set up test path functions first.
CREATE FUNCTION test_path_added() 
RETURNS text AS $$
  import sys
  return str(sys.path[0])
$$ language plpython3u;
SELECT test_path_added();
 test_path_added 
-----------------
 /foo
(1 row)

